### ISSUE TITLE
deprecation in S3 declaration

### ISSUE THREAD

**Issue by aversag** [fivexl/terraform-aws-ecs-events-to-slack#5]
hello,
terraform has updated the S3 declaration, so some options you're using are about to be removed :

```
│ Warning: Argument is deprecated
│ 
│   with module.ecs_to_slack.module.slack_notifications.aws_s3_bucket_object.lambda_package,
│   on .terraform/modules/ecs_to_slack.slack_notifications/main.tf line 126, in resource "aws_s3_bucket_object" "lambda_package":
│  126:   bucket        = var.s3_bucket
│ 
│ Use the aws_s3_object resource instead
│ 
│ (and one more similar warning elsewhere)
```

**Comment by vladimir-cageyv-firi** [comment]
Hello
Yes, right 
We have to update lambda submodule for this that issue

**Comment by Andrey9kin** [comment]
@aversag thanks for reporting. yes, we will release version that will support aws provider v4 shortly

**Comment by Andrey9kin** [comment]
fixed in 0.2.0

**Comment by aversag** [comment]
thank you :) great work
